ckan/logic/action/get.py

```diff
@@ -1184,7 +1184,10 @@ def _group_or_org_show(context, data_dict, is_org=False):
         packages_field = None
 
     include_tags = asbool(data_dict.get('include_tags', True))
-    include_users = asbool(data_dict.get('include_users', True))
+    if asbool(config.get('ckan.auth.public_user_details', True)):
+        include_users = asbool(data_dict.get('include_users', True))
+    else:
+        include_users = asbool(data_dict.get('include_users', False))
```

Comment:
It's not a part of this feature, but maybe it's a time to fix *ValueError*, raised when asbool receives an incorrect value. Maybe, wrap this block with try/except for converting it into ValidationError.  If you try to get **/api/action/group_show?id=GROUP_ID&include_users=tr**, you'll see what i mean